lib/bilrost/worker.js

```diff
@@ -28,6 +28,7 @@ function Worker(topic, subscriber, options, callback, mockBus) {
   this.subscriber = subscriber;
   this.callback = callback;
   this.bus = mockBus || Bus;
+  this.receive = this.receive.bind(this);
```

Comment:
@faygao52 why was this line changed?

Reply:
@shirren I don't know, I didn't change this